.travis.yml

```diff
@@ -11,3 +11,4 @@ before_install:
   - sudo apt-get update -qq
```

Comment:
you can delete these lines, we do not need to install any package anymore. That's super old legacy. It is nice to clean it a little